.github/workflows/windows.yml

```diff
@@ -44,3 +44,24 @@ jobs:
       run: |
         ninja UnitTest
 
+    - name: Prepare artifact
+      shell: pwsh
+      run: |
+    - name: Prepare artifact
+      shell: pwsh
+      run: |
+        mkdir dist
+        copy build\src\Bin\OpenEnroth\OpenEnroth.exe dist\ 
+        copy build\src\Bin\OpenEnroth\*.dll dist\ 
+        if ("${{ matrix.configuration }}" -eq "Debug" -And "${{ matrix.architecture }}" -eq "x86" ) {
+          copy c:\Windows\SysWOW64\msvcp140d.dll dist\ 
+          copy c:\Windows\SysWOW64\vcruntime140d.dll dist\  
+          copy c:\Windows\SysWOW64\ucrtbased.dll dist\
+          copy build\src\Bin\OpenEnroth\*.pdb dist\ 
+        } ElseIf ("${{ matrix.configuration }}" -eq "Debug" -And "${{ matrix.architecture }}" -eq "x64") {
+          copy c:\Windows\System32\msvcp140d.dll dist\ 
+          copy c:\Windows\System32\vcruntime140_1d.dll dist\ 
+          copy c:\Windows\System32\vcruntime140d.dll dist\ 
+          copy c:\Windows\System32\ucrtbased.dll dist\ 
+        }
+        xcopy resources dist\ /E/H/C/I
```

Comment:
Are you adding `Prepare artifact` to regular workflows so that it's tested?

Let's move them out into external sh / bat files, this copy-pasted code will diverge over time otherwise.

Reply:
that was the idea, I just removed that part of the workflow so it still tests what matters, the moving of ucrtbased.dll doesnt really have to happen as a check for pushed code etc, if builds break due to a new version of vcruntime, that can be solved at that point. it only changes once every like 5 years anyways.